### ISSUE TITLE
official site down?

### ISSUE THREAD

**Issue by dylandechant** [angular-ui/angular-google-maps#1842]
http://angular-google-maps.org/ is down for me

**Comment by derfsubterfuge** [comment]
same here... plus I keep getting the 404s for the cluster image: http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclustererplus/images/m1.png

**Comment by getglad** [comment]
+1

**Comment by imjustalf** [comment]
same here, keeps saying too many redirects

**Comment by alecipriani** [comment]
+1

**Comment by scottvincent** [comment]
The googlecode files have been taken down, so the reference to:

http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclustererplus/images/m

needs to be updated to the new github location.

https://raw.githubusercontent.com/googlemaps/js-marker-clusterer/gh-pages/images/m

**Comment by dylandechant** [comment]
If you need to reference the docs you can clone this repo and checkout gh-pages branch. just navigate to the folder and run `python -m SimpleHTTPServer` and navigate to localhost:8000  for mac users, linux should be similar, not sure about windows.

**Comment by derfsubterfuge** [comment]
Has it been permanently taken down?  Was there any notification of this change that I missed?  Why does the library refer to a remote file?  I'm just surprised by this because I updated my version of angular-maps last month.

**Comment by scottvincent** [comment]
@derfsubterfuge Googlecode migrated to github in 2014, and it appears now they're starting to shut things off.

The problem is not only that this project referencing the googlecode.com versions instead of the github versions in the curl.json file, but the actual references in google's github versions are hard coded referencing the missing files still too.

There is a pull request to fix this on their end that needs to be accepted still: https://github.com/googlemaps/js-marker-clusterer/pull/57/files

In conclusion, this is a big mess.

**Comment by aakashjain** [comment]
For now, setting 
`MarkerClusterer.IMAGE_PATH = "https://raw.githubusercontent.com/googlemaps/js-marker-clusterer/gh-pages/images/m";`
in dist/angular-google-maps.js will make things work. 
Sorry if I'm missing something, but if this is what is required, why wait for the PR to be accepted in Google's repository?

**Comment by vatsalpande** [comment]
I needed this directive for my project and the website stopped halfway. Any clue when its going to be up?

**Comment by dotcom9** [comment]
+1

**Comment by scottvincent** [comment]
@aakashjain If I'm not mistaken, if you try to build the source of this project, the dependencies will 404. Even when this project updates them, Google's github project still has the missing images. 

If you use the Dist file of this project, and change that line, it will work.

**Comment by nmccready** [comment]
@scottvincent is correct please see #1844 . I will try my best to get things back up.

**Comment by nmccready** [comment]
#1846

**Comment by nmccready** [comment]
Following #1844 I am in the middle of migrating google mess to my personal repos for the repos I can not find. After that they should be in bower, and npm. I may eventually get to moving them to  meteor too (id rather not).

**Comment by nmccready** [comment]
https://github.com/mahnunchik/markerclustererplus

**Comment by nmccready** [comment]
The build has been fixed. Working on updating the website now.

**Comment by nmccready** [comment]
Site is partially back up. Something is wrong with Dreamhost or github, but the CNAME and or sitemap is causing too many redirects.

Hit this url and it will work. http://angular-ui.github.io/angular-google-maps/index.html

**Comment by nmccready** [comment]
I can't get dream host and the domain angular-google-maps.org to play nice with github. Frankly I am tired of paying for the domain. So I think I am going to just kill dream host.

I have killed the CNAME and sitemap.xml as soon as that and the redirect is from the DNS it starts working again.

**Comment by nmccready** [comment]
So use http://angular-ui.github.io/angular-google-maps
